src/App.tsx

```diff
@@ -0,0 +1,15 @@
+import React, { Component } from "react";
+import Counter from "/home/niv/myflipcounter/myflipcounter/src/FlipCounter.tsx";
+
+import "./App.css";
+class App extends Component {
+  render() {
+    return (
+      <div id="body">
+        <Counter />
```

Comment:
In the task it is written, that 

- The name of the component is FlipCounter
- The component receives one prop `value` with the type Number

And you have done one big component with all the functionality

The primary idea of React is [composition of components](https://reactjs.org/docs/composition-vs-inheritance.html) and each of the components should take one responsibility.

So you need to use composition and divide components to the one, with the input and add button. It will be responsible for taking user's input and passing it to FlipCounter.

And FlipCounter is accepting a value to render though props and responsible for the view of this component.

So FlipCounter should receive one prop: `value`